views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -139,6 +141,20 @@ define([
             interaction.attr('patternMask', attrValue);
         };
 
+        callbacks.expectedLength = function(interaction, attrValue){
+            var newValue = parseInt(attrValue,10);
+            if(! isNaN(newValue)){
+                interaction.attr('expectedLength', attrValue);
+            }
+        };
```

Comment:
ok. I'll see. Is there's a way to use the rendering template and reload to display the changes ?